spec/requests/api/v1/access_token_spec.rb

```diff
@@ -1,38 +1,31 @@
-# require 'rails_helper'
-#
-#
-# describe 'GET /api/v1/access_token/:id' do
-#   before do
-#     WebMock.allow_net_connect!
-#   end
-#
-#   after do
-#     WebMock.disable_net_connect!
-#   end
-#   it 'returns new access_token for specific user' do
-#
-#     user_1 = create(:user, id: 1, refresh_token: ENV['SPOTIFY_REFRESH_TOKEN'])
-#     user_2 = create(:user, id: 2, refresh_token: "12345")
-#
-#     get "/api/v1/access_token/#{user_1.id}"
-#
-#     expect(response.status).to eq(200)
-#
-#     data = JSON.parse(response.body, symbolize_names: true)
-#
-#     expect(data).to have_key(:access_token)
-#     expect(data[:access_token].length).to eq(163)
-#   end
-#
-#   it 'returns 404 if user not found' do
-#
-#     get "/api/v1/access_token/1000"
-#
-#     expect(response.status).to eq(404)
-#
-#     data = JSON.parse(response.body, symbolize_names: true)
-#
-#     expected_error = {error: "Could not find record with user_id: 1000"}
-#     expect(data).to eq(expected_error)
-#   end
-# end
+require 'mock_helper'
+
+describe 'GET /api/v1/access_token/:id' do
+  it 'returns new access_token for specific user' do
```

Comment:
@fentontaylor ... I don't fully understand why we are stubbing the access_token endpoint. Maybe we can talking about this tomorrow. 
As I'm reading the test, we're basically just testing that our access_token endpoint returns the stub information... testing if the stub works rather than testing if Spotify is indeed returning an access_token. I've typically use Webmock to stub part of a feature, not the end product. So example, stubbing out an API's response that used for another API or is dramatically reformatted and changed value is returned as the final response, if that makes sense.

Reply:
Thinking about it, perhaps a better test is call the access_token twice, saving each call to a variable. Then, testing that the two variables do not equal one another. Thereby testing, an unique access_token is return each time the access_token endpoint is called.

Reply:
Thank you for explaining the webmock for me this morning. Looks like we're not mocking out the end product for Spotify access_token endpoint which is good :D